src/environments/environment.prod.ts

```diff
@@ -20,3 +20,24 @@ export const environment: Environment = {
 
   itemPlatformId: 'algorea_backend',
 };
+
+export const presets: Record<string, Partial<Environment>> = {
+  example: {
+    defaultActivityId: '1625159049301502151', // Motif Art
+    defaultTitle: 'Example app',
+    authType: 'tokens',
+    itemPlatformId: 'Example',
+  },
+  demo: {
+    defaultActivityId: '1352246428241737349', // SNT
+    defaultTitle: 'Demo app'
+  },
+};
+
+export function getPresetNameByDomain(domain: string): string | null {
+  switch (domain) {
+    case 'example.algorea.org': return 'example';
```

Comment:
I think it would make sense to include the protocol as typically we may have the "http" where cookies cannot be used and "https" using cookies.

Reply:
fixed